libs/math/include/math/quat.h

```diff
@@ -126,6 +126,43 @@ class MATH_EMPTY_BASES TQuaternion :
     constexpr static TQuaternion MATH_PURE fromAxisAngle(const TVec3<A>& axis, B angle) {
         return TQuaternion(std::sin(angle * 0.5) * normalize(axis), std::cos(angle * 0.5));
     }
+
+    // constructs a quaternion from orig to dest. orig and dest must be normalized.
+    template<typename A, typename B, typename = enable_if_arithmetic_t<A, B>>
+    constexpr static TQuaternion MATH_PURE fromOrigToDest(const TVec3<A>& orig, const TVec3<B>& dest) {
```

Comment:
I don't like the name because it uses abbreviated words, which we are trying to avoid in our APIs. I agree that to be consistent with the other APIs it should be `fromXXX`.  Maybe simply `fromVectorAtoB`?

Reply:
It could be just `from`. Or `rotation`.

Reply:
Yes, but there are some constraints on the name, I think:
1) `fromXXX()` is more in line with `fromAxisAngle()` for instance. It shows it's a factory.
2) We want a name that implies the direction of the transform w.r.t. the parameters. e.g. `from()` or `rotation()` are unclear about is it from param1 to param2 or vice-versa.
So I thought `fromVectorAtoB(vec a, vec b)` was clearer, even if it sounds strange.

Reply:
`fromVectorAtoB()` reads a like a repetition of the arguments to me. `fromDirectedRotation()` or similar would make more sense I think (I guess this implies a src/dst order, but that doesn't seem far fetched to make that assumption).

Reply:
Change name to `fromDirectedRotation`.